Mathlib/Data/FunLike/Basic.lean

```diff
@@ -125,6 +125,10 @@ injective coercion to functions from `α` to `β`.
 
 This typeclass is used in the definition of the homomorphism typeclasses,
 such as `ZeroHomClass`, `MulHomClass`, `MonoidHomClass`, ....
+
+Lean will try to synthesize non-dependent `FunLike` typeclasses first by default. Use the command
+`attribute [local instance high] FunLike.hasCoeToFun` if you want dependent `FunLike` typeclasses
+to be synthesized.
```

Comment:
Did any files end up needing this?
```suggestion
`attribute [local instance high] FunLike.hasCoeToFun` for better performance when you are using lots of dependent `FunLike` typeclasses.
```
to make it clear that this is only a performance concern.

Reply:
I added it in `Mathlib.Geometry.Manifold.VectorBundle.SmoothSection`. It was the file that slowed down significantly in the first benchmark.

Reply:
Note it down there too, I think